### PR TITLE
Docs for new Team Billing

### LINKED ISSUE #2216: Document new team billing
We introduced new way to manage team billing for project teams with gitpod-io/gitpod#8041.   
This needs to be documented to explain the difference is between the 2 subscription models.

The benefit of the new team billing mechanism (compared to the older "[Team Plans](https://www.gitpod.io/docs/teams)" in your personal settings), is that adding/removing members from a project team automatically adds/removes members from the billing subscription. The selection and price per seat of the 2 plans is identical.

The new project team billing will auto-subscribe every existing member of the project team when team billing is set up. This includes the owner of the team (unlike the older team plans, whose owners are not auto-subscribed). 

For more control over who participates in the team plan billing, you can still use the older Team Plans, but you will have to manage each seat manually.

### PR DESCRIPTION
**Pull request by jldec**
## Description
- Document new **Team Billing** at /docs/team-billing
- Improve **Team Plans**
- Link from **Teams & Projects**

## Related Issue(s)
Fixes #2216

### Preview
- https://deploy-preview-2256--gitpod-kumquat.netlify.app/docs/team-billing
- https://deploy-preview-2256--gitpod-kumquat.netlify.app/docs/teams
- https://deploy-preview-2256--gitpod-kumquat.netlify.app/docs/teams-and-projects

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2256"></a>

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2256"></a>

### CODE REVIEW

File: gitpod/docs/team-billing.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+title: Team Billing
+---
+
+<script context="module">
+  export const prerender = true;
+</script>
+
+# Team Billing
+
+> Team Billing is part of [Teams & Projects](/docs/teams-and-projects), currently in [Beta](/docs/references/gitpod-releases).
```
Comment (jankeromnes) [inline gitpod/docs/team-billing.md:12]: Oh, are Teams & Projects really still in Beta? Or just Team Billing? ~~(If the latter, maybe you could clarify with "and currently in Beta"?)~~

EDIT: Answered my own question by reading further down. ✔️

File: gitpod/docs/team-billing.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+> **NOTE**  
+> This first iteration of Team Billing improves Team Plans by:
+>
+> 1. Auto-subscribing new members when they join or leave the team.
+> 2. Pro-rata billing for new members who join in the middle of a billing cycle.
```
Comment (jankeromnes) [inline gitpod/docs/team-billing.md:18]: ❤️

File: gitpod/docs/team-billing.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+>
+> If you prefer to continue to manage membership of [**Team Plans**](/docs/teams) manually, you can.
```
Comment (jankeromnes) [inline gitpod/docs/team-billing.md:20]: I wonder if we should double-clarify by calling them "legacy Team Plans" 💭 (I think there is going to be confusion anyway between Team Plans and Team Billing, so maybe being overly explicit can be helpful?)

I know that you're already using two different terms here (Team Billing vs Team Plans), but there are also still two available "plans" in this new Team Billing. 😅

File: gitpod/docs/teams-and-projects.md [hunk 4]
```diff
@@ -11,15 +11,14 @@ title: Teams & Projects
...
 
 ## Teams
 
-You can use teams to group and manage one or more projects and invite team members.
-Users with access to a team can view branches, prebuilds, and configuration for all projects within a team as well as open new workspaces directly from a branch or a prebuild from the dashboard.
+Teams are used to share projects amongst team members. All users with access to a team can view branches and prebuilds for the projects in the team. They can also open new workspaces directly from a branch in the dashboard.
+
+Owners of a team can manage other members, and setup [Team Billing](/docs/team-billing).
 
 ### View teams
```
Comment (jankeromnes) [inline gitpod/docs/teams-and-projects.md:24]: In "Add members to a team", I wonder if it's worth mentioning that you can also reset the team invite URL. 💭

**Review by jankeromnes — APPROVED** [review]
This is fantastic 😍 many thanks @jldec for these awesome documentation improvements! 💯

Every page is spot-on 🎯 helpful and quite nice to read. 📖

I've left a few optional comments here and there, but please feel free to dismiss any of them and merge this as is. It's already perfect. ✨

File: gitpod/docs/teams-and-projects.md (hunk 4 above)
Comment (jldec) [reply gitpod/docs/teams-and-projects.md:24]: good idea 👍

File: gitpod/docs/team-billing.md (hunk 3 above)
Comment (jldec) [reply gitpod/docs/team-billing.md:20]: Reluctant to add another word like legacy.

**Pull request merged**